package.json

```
@@ -0,0 +1,61 @@
+{
+  "name": "jslab_frontend_part",
+  "version": "1.0.0",
+  "description": "set up frontend",
```

This description doesn't make sense
```suggestion
  "description": "",
```